package_modules/apt_get

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import re
+
+dpkg_cmd = "/usr/bin/dpkg"
+dpkg_deb_cmd = "/usr/bin/dpkg-deb"
+dpkg_query_cmd = "/usr/bin/dpkg-query"
+dpkg_output_format = "Name=${Package}\nVersion=${Version}\nArchitecture=${Architecture}\n"
+dpkg_status_format = "Status=${Status}\n" + dpkg_output_format
+
+apt_get_options = ["-o", "Dpkg::Options::=--force-confold", "-o", "Dpkg::Options::=--force-confdef", "-y", "--force-yes"]
+apt_get_cmd = "/usr/bin/apt-get"
+
+os.environ['DEBIAN_FRONTEND'] = "noninteractive"
+os.environ['LC_ALL'] = "C"
+
+if (os.environ.get('CFENGINE_TEST_DPKG_CMD')):
+    dpkg_cmd = os.environ['CFENGINE_TEST_DPKG_CMD']
+if (os.environ.get('CFENGINE_TEST_DPKG_DEB_CMD')):
+    dpkg_deb_cmd = os.environ['CFENGINE_TEST_DPKG_DEB_CMD']
+if (os.environ.get('CFENGINE_TEST_DPKG_QUERY_CMD')):
+    dpkg_query_cmd = os.environ['CFENGINE_TEST_DPKG_QUERY_CMD']
+if (os.environ.get('CFENGINE_TEST_APT_GET_CMD')):
+    apt_get_cmd = os.environ['CFENGINE_TEST_APT_GET_CMD']
+
+def get_package_data():
+    pkg_string = ""
+    for line in sys.stdin:
+        result = re.match("^File=(.*)", line)
```

Comment:
Python documentation states that you don't have to, it will pre-compile for you for "small" programs. It makes no mention of what a small program is, but I doubt any of these will exceed the limit.